docs/source/qmrflow_mtr.rst

```diff
@@ -0,0 +1,160 @@
+MTR: Magnetization transfer ratio
+==================================
+
+.. |br| raw:: html
+
+  <br />
+
+Usage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: console
+
+   nextflow run mtrflow_BIDS.nf [OPTIONAL_ARGUMENTS] (--root)
+
+Description
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``--root=/path/to/[root]``                    Root folder containing multiple subjects
+
+Container requirements 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have Docker installed, enabling docker option will make use of the 
+following Docker images to execute processes: 
+
+  - qmrlab/minimal (https://hub.docker.com/repository/docker/qmrlab/minimal)
+                    594MB extends to 1.5GB
+                    Built at each qMRLab release.  
+                    Minimum version requirement: v2.3.1 
+  - qmrlab/antsfsl (https://hub.docker.com/repository/docker/qmrlab/antsfsl)
+                    374MB extends to 1.2GB                      
+                    Dockerfile is available at qMRLab/qMRflow.
+
+Local installation requirements 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Unless the docker option is enabled in the `nextflow.config`, the following
+dependencies must be installed and added to the system path: 
+
+  * ANTs registration (https://github.com/ANTsX/ANTs)
+  * FSL (https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/)
+  * Octave/MATLAB (https://www.gnu.org/software/octave/,https://www.mathworks.com/)
+  * qMRLab > v2.3.1 (https://qmrlab.org)
+  * git
+
+Folder organization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: https://upload.wikimedia.org/wikipedia/commons/d/de/BIDS_Logo.png
+  :width: 200
+
+.. code-block:: console
+
+    [root]
+    ├── sub-01
+    │   └── anat
+    │       ├── sub-01_acq-MTon_MTR.nii.gz
+    |       ├── sub-01_acq-MTon_MTR.json
+    │       ├── sub-01_acq-MToff_MTR.nii.gz
+    │       └── sub-01_acq-MToff_MTR.json
+    └── sub-02
+            └── anat
+                ├── sub-02_acq-MTon_MTR.nii.gz
+                ├── sub-02_acq-MTon_MTR.json
+                ├── sub-02_acq-MToff_MTR.nii.gz
+                └── sub-02_acq-MToff_MTR.json
+
+.. note:: This workflow can use a subset of the MTsat (`MTS`) data.
+
+Optional arguments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+--platform                      ``["octave"/"matlab"]`` Platform choice.
+--qmrlab_dir                    ``["/path/to/qMRLab" OR null]`` Absolute path to the qMRLab's |br|
```

Comment:
is there a reason to chose underscore instead of the more popular dash for separating multiple words on flags? I find dash easier to write (no shift required)

Reply:
Thanks! https://github.com/qMRLab/qMRFlow/issues/4